packages/storybook/cxl-ui/cxl-paywall/cxl-paywall-layout=2c-l.stories.js

```diff
@@ -104,13 +104,12 @@ const Template = ({ _count, _limit, delay, opacity, subscribed }) => html`
         <label>Playbook</label>
         <h1 class="entry-title">Choose a traditional SaaS go-to-market strategy</h1>
         <div class="entry-byline">
-          <span class="progress statement playbook-completion-rate"
-            >Completed 4 steps of 8 in total</span
-          >
-          <vaadin-progress-bar value="0.50">Completed 4 steps of 8 in total</vaadin-progress-bar>
-          <section class="course-meta course-enrolment">
-            <div class="status in-progress">In Progress</div>
-          </section>
+          <label for="cxl-playbook-progress-bar-${postId}"></label>
+          <cxl-playbook-progress-bar
+            id="cxl-playbook-progress-bar-${postId}"
+            listen-on-closest="article"
+          ></cxl-playbook-progress-bar>
+          <!-- [listen-on-closest] fixes race condition vs article classes computation -->
```

Comment:
This doesn't seem to be connected to Vaadin upgrade? We're refactoring towards a whole new element here.

Reply:
@lkraav We sort of snuck this in there. It would be easy to remove and separate into a new PR.

Reply:
I removed the commit.